cluster/gce/util.sh

```diff
@@ -771,39 +771,78 @@ function create-nodes() {
 # - NODE_INSTANCE_PREFIX
 # - PROJECT
 # - ZONE
+# - AUTOSCALER_MAX_NODES
+# - AUTOSCALER_MIN_NODES
+function create-cluster-autoscaler-mig-config() {
+
+  # Each MIG must have at least one node.
+  if $((${AUTOSCALER_MIN_NODES} < ${NUM_MIGS})); then
+    echo "AUTOSCALER_MIN_NODES must be greater or equal ${NUM_MIGS}"
+    exit 2    
+  fi
+
+  # Each MIG must have at least one node.
+  if $((${AUTOSCALER_MAX_NODES} < ${NUM_MIGS})); then
+    echo "AUTOSCALER_MAX_NODES must be greater or equal ${NUM_MIGS}"
+    exit 2    
+  fi
+
+  local min_nodes_left=$((${AUTOSCALER_MIN_NODES}-${NUM_MIGS}))
```

Comment:
I don't understand how do you compute size of each MIG. Please add extensive comments to both the function and the code!